pkg/tempo/config.go

```diff
@@ -0,0 +1,192 @@
+package tempo
+
+import (
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/spf13/viper"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/exporter/otlpexporter"
+	"go.opentelemetry.io/collector/processor/batchprocessor"
+	"go.opentelemetry.io/collector/processor/queuedprocessor"
+	"go.opentelemetry.io/collector/receiver/jaegerreceiver"
+	"go.opentelemetry.io/collector/receiver/otlpreceiver"
+	"go.opentelemetry.io/collector/receiver/zipkinreceiver"
+)
+
+// Config controls the configuration of the Tempo trace pipeline.
+type Config struct {
+	// Whether the Tempo subsystem should be enabled.
+	Enabled bool `yaml:"-"`
+
+	RemoteWrite RWConfig `yaml:"remote_write"`
+
+	// Receivers: https://github.com/open-telemetry/opentelemetry-collector/tree/master/receiver
+	Receivers map[string]interface{} `yaml:"receivers"`
+}
+
+// RWConfig controls the configuration of exporting to Grafana Cloud
+type RWConfig struct {
+	URL       string                 `yaml:"url"`
+	Insecure  bool                   `yaml:"insecure"`
+	BasicAuth *BasicAuthConfig       `yaml:"basic_auth,omitempty"`
```

Comment:
Can you import this from [prometheus/common](https://github.com/prometheus/common/blob/f39dfa2b000545b3a763b844012958c275a62266/config/http_config.go#L41) instead?